Makefile

```diff
@@ -82,6 +100,43 @@ yaml-lint: ## run YAML Lint
 	@echo "----------------------------"
 	yamllint -c .yamllint ./config.yaml ./config
 
+.PHONY: db
+db:
+	@echo "Generating DB Release Yaml... "
+	mkdir -p releases && cd config && ko resolve -f 00-init  > ../releases/db.yaml
+	@echo "-----------------------------------------"
+
+.PHONY: db-migration
+db-migration:
+	@echo "Generating DB-Migration Release Yaml... "
+	mkdir -p releases && cd config && ko resolve -f 01-db  > ../releases/db-migration.yaml
+	@echo "----------------------------"
+
+.PHONY: api-k8s
+api-k8s:
+	@echo "Generating K8s Specific Api Release Yaml... "
+	mkdir -p releases && cd config && ko resolve -f 02-api -f 04-kubernetes/40-api-ingress.yaml  > ../releases/api-k8s.yaml
+	@echo "-----------------------------------------"
+
+.PHONY: api-openshift
+api-openshift:
+	@echo "Generating Openshift Specific Api Release Yaml... "
+	mkdir -p releases && cd config && ko resolve -f 02-api -f 04-openshift/40-api-route.yaml > ../releases/api-openshift.yaml
+	@echo "-----------------------------------------"
+
+.PHONY: ui-k8s
+ui-k8s:
+	@echo "Generating K8s Specific UI Release Yaml... "
+	mkdir -p releases && cd config && ko resolve -f 03-ui -f 04-kubernetes/42-ui-ingress.yaml > ../releases/ui-k8s.yaml
```

Comment:
Should we keep ingress, as it depends on whether the controller is installed?

Reply:
I think you are right, we can ignore the ingress

Reply:
Can you please remove ingress?